en/guide/index.md

```diff
@@ -22,57 +22,57 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
 
 Nuxt.js includes the following to create a rich web application development:
 
 - [Vue 2](https://github.com/vuejs/vue)
-- [Vue-Router](https://github.com/vuejs/vue-router)
+- [Vue Router](https://github.com/vuejs/vue-router)
 - [Vuex](https://github.com/vuejs/vuex) (included only when using the [store option](/guide/vuex-store))
-- [Vue-Meta](https://github.com/declandewet/vue-meta)
+- [Vue Meta](https://github.com/declandewet/vue-meta)
 
-A total of only **57kb min+gzip** (53kb with vuex).
+A total of only **57kb min+gzip** (53kb with Vuex).
 
-Under the hood we use [Webpack](https://github.com/webpack/webpack) with [vue-loader](https://github.com/vuejs/vue-loader) and [babel-loader](https://github.com/babel/babel-loader) to bundle, code-split and minify your code.
+Under the hood we use [webpack](https://github.com/webpack/webpack) with [vue-loader](https://github.com/vuejs/vue-loader) and [babel-loader](https://github.com/babel/babel-loader) to bundle, code-split and minify your code.
 
 ## Features
 
-- Write Vue Files
+- Write Vue Files (`*.vue`)
 - Automatic Code Splitting
 - Server-Side Rendering
 - Powerful Routing System with Asynchronous Data
 - Static File Serving
 - ES6/ES7 Transpilation
 - Bundling and minifying of your JS & CSS
-- Managing `<head>` element (title, meta...)
+- Managing `<head>` element (`<title>`, `<meta>`, etc.)
 - Hot module replacement in Development
-- Pre-processor: SASS, LESS, Stylus, etc
+- Pre-processor: Sass, Less, Stylus, etc.
 - HTTP/2 push headers ready
 - Extending with Modular architecture
 
 ## Schema
 
-This schema shows what is called by nuxt.js when the server is called or when the user navigate through the app via `<nuxt-link>`:
+This schema shows what is called by Nuxt.js when the server is called or when the user navigate through the app via `<nuxt-link>`:
 
 ![nuxt-schema](/nuxt-schema.png)
 
 ## Server Rendered (Universal SSR)
 
-You can use nuxt.js as a framework to handle all the UI rendering of your project.
+You can use Nuxt.js as a framework to handle all the UI rendering of your project.
 
-When launching `nuxt`, it will start a development server with hot-reloading and [vue-server-renderer](https://ssr.vuejs.org/en/) configured to automatically server-render your application.
+When launching `nuxt`, it will start a development server with hot-reloading and [Vue Server Renderer](https://ssr.vuejs.org/en/) configured to automatically server-render your application.
```

Comment:
Because in french we translate “hot-reload” by « rechargement à chaud » this have no link with a “package” name so it made more sens to use the product name for “Vue Server Renderer”. In addition, you link the official documentation and not the npm/github link so you refer to the “Vue Server Renderer” product and not “vue-server-renderer” package name.